sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/FluxUtil.java

```diff
@@ -199,6 +200,21 @@ public static boolean isFluxByteBuf(Type entityType) {
             .flatMap(serviceCall);
     }
 
+    /**
+     * Converts the incoming content to Mono.
+     *<p>
+     * If the response {@link Response#value() value} is empty, an empty Mono is returned.
+     *</p>
+     * <p><strong>Code samples</strong></p>
+     * {@codesnippet com.azure.core.implementation.util.fluxutil.toMono}
+     *
+     * @param response whose {@link Response#value() value} is to be converted
+     * @return The converted {@link Mono}
+     */
+    public static <T> Mono<T> toMono(Response<T> response) {
```

Comment:
Somewhat redundant JavaDoc (and especially the code snippet) as this is an implementation class and therefore will never be generated.